src/hera/expr/_sprig.py

```diff
@@ -0,0 +1,16 @@
+from typing import Any
```

Comment:
Perhaps documentation with a link to http://masterminds.github.io/sprig/ would be helpful